microservice

```diff
@@ -0,0 +1,12 @@
+# microservice architecture
+
+This File lists all the microservices that are used in the project.
+
+- User Service
+- Authentication Service
+- Course Service
+  - Course Catalog Service
+  - Course Enrollment Service
+  - Course Rating Service
+  - Course Assessment Service
+- Company Service
```

Comment:
Should there be at least a small description about each services and what its purpose is?